packages/tocco-util/src/appFactory/appFactory.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import {Provider} from 'react-redux'
+import {addLocaleData} from 'react-intl'
+import {IntlProvider} from 'react-intl-redux'
+
+import de from 'react-intl/locale-data/de'
+import en from 'react-intl/locale-data/en'
+import fr from 'react-intl/locale-data/fr'
+import it from 'react-intl/locale-data/it'
+
+import {LoadMask} from 'tocco-ui'
+
+import storeFactory from '../storeFactory'
+import intl from '../intl'
+import externalEvents from '../externalEvents'
+
+export const createApp = (name, content, reducers, sagas, input, events, actions, publicPath) => {
+  try {
+    setWebpacksPublicPath(publicPath)
+    const initialState = getIntialState(input)
+
+    if (events) {
+      externalEvents.registerEvents(events)
+    }
+
+    const store = storeFactory.createStore(initialState, reducers, sagas)
+    const initIntlPromise = setupIntl(input, store, name)
+
+    dispatchActions(actions, store)
+    const component = getAppComponent(store, initIntlPromise, name, content)
+
+    return {
+      renderComponent: component,
+      store,
+      methods: {
+        setLocale: locale => intl.setLocale(store, name, locale)
+      }
+    }
+  } catch (e) {
+    console.log('Error creating react application: ', e)
```

Comment:
maybe we should add a logging utility which prevents from null pointers if console is not defined